plug.py

```diff
@@ -312,6 +312,29 @@ def notify_newdata(data, is_beta):
     return error
 
 
+def notify_journal_entry_cqc(cmdr, is_beta, entry, state):
+    """
+    Send a journal entry to each plugin.
+    :param cmdr: The Cmdr name, or None if not yet known
+    :param entry: The journal entry as a dictionary
+    :param state: A dictionary containing info about the Cmdr, current ship and cargo
+    :param is_beta: whether the player is in a Beta universe.
+    :returns: Error message from the first plugin that returns one (if any)
+    """
+
+    error = None
+    for plugin in PLUGINS:
+        journal_entry = plugin._get_func('journal_entry_cqc')
```

Comment:
probably rename this cqc_callback? or something a bit less opaque than `journal_entry`

Reply:
Probably yes but may be time has come to define custom prefix for such kind of functions (https://github.com/EDCD/EDMarketConnector/issues/1226) ?

Reply:
I meant the variable, not the plugin side function name